internal/services/appservice/helpers/linux_web_app_schema.go

```diff
@@ -1174,6 +1158,10 @@ func (s *SiteConfigLinux) ExpandForUpdate(metadata sdk.ResourceMetaData, existin
 	}
 
 	if metadata.ResourceData.HasChange("site_config.0.auto_heal_setting") {
+		expanded.AutoHealEnabled = pointer.To(false)
```

Comment:
This is already `false` from the top of the function, can we check the logic here and remove one of these?